docs/getting-started/creating-your-first-bot.mdx

```diff
@@ -166,15 +166,23 @@ is a module that we will use to load the env file. You installed this with `pip
 Next, we load the env file with `load_dotenv()`.
 
 ```py
-bot = discord.Bot(debug_guilds=[...])
+bot = discord.Bot()
 ```
 
 In this line, we create a new instance of [`discord.Bot`](https://docs.pycord.dev/en/master/api.html#discord.Bot).
-In this object, we pass a `debug_guilds` argument, which is a list of guild IDs that the bot's application
-commands will appear in. This is beneficial for testing purposes when we are testing a new command
-or two and don't want everyone to be able to use it. If debug guilds are not specified, all of the 
-servers the bot is in will have access to your slash commands.
+In this object, we can pass various parameters for configuration purposes, such as `owner_ids`
+and [`intents`](https://docs.pycord.dev/en/stable/api.html?highlight=get_message#discord.Intents).
+
+:::caution
 
+Considering that Discord no longer takes 1 hour to update Slash Commands,
+we do not recommend passing `debug_guilds` as parameter because
+it will prevent any global Slash Command to be registered. Instead,
+set [`guild_ids`](https://docs.pycord.dev/en/stable/api.html?highlight=get_message#discord.SlashCommand.guild_ids)
+for the [Application Command](https://docs.pycord.dev/en/stable/api.html?highlight=get_message#application-commands)
+that you intend to test.
+
+:::
```

Comment:
IDT this section is needed. The `debug_guilds` param can still be used if someone wants a guild-only bot, we just don't want to recommend it as a default for new users anymore.
```suggestion
```